test/utils/InstanceTest.sol

```diff
@@ -238,6 +201,7 @@ abstract contract InstanceTest is HyperdriveTest {
 
         // Ensure that refunds are handled properly.
         if (config.enableBaseDeposits && asBase && isBaseETH) {
+            // Ensure that alice's ETH balance only decreased by the contribution.
```

Comment:
```suggestion
            
            // Ensure that alice's ETH balance only decreased by the contribution.
```

Reply:
We discussed this offline and resolved this.